swig/CMakeLists.txt

```diff
@@ -1,17 +1,16 @@
 FIND_PACKAGE(SWIG REQUIRED)
 INCLUDE(${SWIG_USE_FILE})
 
-#FIXME: duplicates parent
```

Comment:
To clarify, someone removed this python dependency from the swig subdir CMakeLists in response to the same piece of code being unnecessarily copypasted from libqasm into the main CMakeLists. Now that those libqasm's CMakeLists are more or less sane and just require an `add_subdirectory()` I'd removed the unnecessarily copypasted bit from the main CMakeLists, so stuff broke. I opted to just go back to the original state before libqasm was pulled in.

Reply:
Wouter removed this after a painfully long process to debug a failed CI due to this code selecting a different python interpreter than the parent one. So let's see if with the modifications in the other files this now passes CI.

Reply:
That's... odd. Looks like it passes again now though.

Reply:
Yes, that's indeed odd. You can check the PR #321 if you are interested in that problem. Good that it passes again, I will merge.